consensus/src/chained_bft/event_processor_fuzzing.rs

```diff
@@ -127,9 +127,12 @@ fn create_node_for_fuzzing() -> EventProcessor<TestPayload> {
     // TODO: have two different nodes, one for proposing, one for accepting a proposal
     let proposer_election = Box::new(RotatingProposer::new(vec![signer.author()], 1));
 
+    let pending_votes = PendingVotes::new();
```

Comment:
minor: it seems that we should never try to keep pending votes across epochs. In this case there is probably no need to pass the `pending_votes` to `EventProcessor`, it is always an empty one anyway. Let `EventProcessor` create it by itself?